validator/constants.py

```diff
@@ -43,9 +43,9 @@
 with open(os.path.join(os.path.dirname(__file__), 'app_versions.json')) as avs:
     APPROVED_APPLICATIONS = json.load(avs)
 
-# Add the '*' fake max version. Used specifically for WebExtensions, see
-# https://github.com/mozilla/amo-validator/issues/390.
+# Add the '*' fake max version for Desktop and Android.
 APPROVED_APPLICATIONS['1']['versions'].append('*')
+APPROVED_APPLICATIONS['7']['versions'].append('*')
```

Comment:
I know you are just copying the last person but this was a terrible approach IMO. Instead, what if you did it like this:
````python
for app_id, config in APPROVED_APPLICATIONS.items():
    if config['name'] in ('Firefox', 'Android'):
        APPROVED_APPLICATIONS[app_id]['versions'].append('*')
````